app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +31,26 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/39d86fb759bdab1db76b818f1155db0332b36239/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L36"></span>Variable ‘text’ should be declared final.</td>
    </tr>
  </tbody>
</table>